src/backend/distributed/commands/truncate.c

```diff
@@ -372,9 +372,10 @@ LockTruncatedRelationMetadataInWorkers(TruncateStmt *truncateStatement)
 	}
 
 	RangeVar *rangeVar = NULL;
+	List *referencingRelationIds = NIL;
 	foreach_ptr(rangeVar, truncateStatement->relations)
 	{
-		Oid relationId = RangeVarGetRelid(rangeVar, NoLock, false);
+		relationId = RangeVarGetRelid(rangeVar, NoLock, false);
```

Comment:
Minor: mostly declare variables closer to where they are used. For `relationId` the old way conforms better to the conventions